pegasus/sites.v3/code.org/views/homepage_hero.haml

```diff
@@ -188,6 +191,13 @@
     showQuote(lastImage, currentImage);
   }
 
+  function clickHero() {
+    heroTarget = $("#fullwidth"+currentImage).attr("data-click-url");
+    if (heroTarget !== null && heroTarget !== undefined) {
+      window.location.href = heroTarget;
+    }
+  }
+
   $.ready()
   {
```

Comment:
Good point.. addressed in https://github.com/code-dot-org/code-dot-org/pull/12490